backend/src/routes/register.routes.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import { registerController } from '../controllers/registerController';
+import { validateUser } from '../middlewares/userValidator';
+
+const cors = require('cors');
+
+const router = express.Router();
+
+router.use(cors());
+router.use(bodyParser.json());
```

Comment:
@marcianszlovak these will not be needed, since these are configured under `/api` where this should be moved